### ISSUE TITLE
support for macOS arm64

### ISSUE THREAD

**Issue by kuyoonjo** [idealvin/co#170]
Compiling failed for macos arm64:

```
$ xmake
checking for architecture ... arm64
checking for Xcode directory ... /Applications/Xcode.app
checking for Codesign Identity of Xcode ... no
checking for SDK version of Xcode for macosx (arm64) ... 11.3
checking for Minimal target version of Xcode for macosx (arm64) ... 11.5
checking for HAS_EXECINFO_H ... ok
[  2%]: compiling.release src/co/context/context.S
[  2%]: compiling.release src/__/stack_trace.cc
[  2%]: compiling.release src/__/stack_trace_win.cc
[  2%]: compiling.release src/hash/base64.cc
[  2%]: compiling.release src/time_win.cc
[  2%]: compiling.release src/json.cc
[  2%]: compiling.release src/hash/url.cc
[  2%]: compiling.release src/hash/murmur_hash.cc
[  2%]: compiling.release src/hash/crc16.cc
[  2%]: compiling.release src/hash/md5.cc
[  2%]: compiling.release src/fastring.cc
[  2%]: compiling.release src/path.cc
error: <instantiation>:10:9: error: unknown directive
        .type _tb_context_make, %function
        ^
src/co/context/context_arm64.S:67:1: note: while in macro instantiation
function tb_context_make, export=1
^
<instantiation>:11:9: error: unknown directive
        .hidden _tb_context_make
        ^
src/co/context/context_arm64.S:67:1: note: while in macro instantiation
function tb_context_make, export=1
^
<instantiation>:1:1: error: unknown directive
.size tb_context_make, . - tb_context_make
^
src/co/context/context_arm64.S:117:1: note: while in macro instantiation
endfunc
```

**Comment by idealvin** [comment]
@kuyoonjo 
https://www.zhihu.com/people/vin.cc 去这里给我发条私信，进微信群.

**Comment by idealvin** [comment]
@kuyoonjo zh 已经回复了..

**Comment by idealvin** [comment]
fixed in #173
